src/ng/directive/input.js

```diff
@@ -1711,7 +1716,7 @@ var CONSTANT_VALUE_REGEXP = /^(true|false|\d+)$/;
  *
  * @element input
  * @param {string=} ngValue angular expression, whose value will be bound to the `value` attribute
- *   of the `input` element
+ * and `value` property of the `input` or `textarea` element
```

Comment:
I don't think `input` was meant as `<input>` (it would be wrong). Maybe we should just say `element`, since it can be `input`, `textarea`, `option` (anything actually).